HTTPServerProject.Tests/HTTPServerReadTests/TestReadStreams.cs

```diff
@@ -0,0 +1,104 @@
+using HTTPServerRead.Streams;
+
+namespace HTTPServerReadTests.Streams;
+
+public class UnitTestsForReadStreams
```

Comment:
This looks like a test for `TestReadStreams` - you don't need to test test doubles, you can just use them in your tests.

Reply:
Sounds good.